generated/stackstate_api/api/openapi.yaml

```diff
@@ -444,46 +444,6 @@ paths:
       summary: Get a monitor
       tags:
       - monitor
-    patch:
```

Comment:
OpenAPI master has this
https://gitlab.com/stackvista/platform/stackstate-openapi/-/blob/master/spec/monitors.api.yaml#L39